deploy/install.go

```diff
@@ -640,7 +640,7 @@ func (dp *DeployProcess) WriteConfigMapManifest() {
 
 	settings := dp.cfg.AllSettings()
 
-	logLevel := "info"
+	logLevel := "debug"
```

Comment:
Why do we want this? The default is usually info, debug can be set through the ConfigMap if additional logs are required.

Reply:
I noticed that the drivers and the Helm deployment of Authorization are debug by default so I thought we can be consistent.

Reply:
I see. I don't think we have a set standard, we can leave it in to be consistent.